### ISSUE TITLE
Contacts are always shown as offline

### ISSUE THREAD

**Issue by Legodev** [EionRobb/icyque#9]
For me all my contacts are shown as offline even while talking to them. When i use the WebIcq they are shown as online.

**Comment by hitman72ita** [comment]
yes, from a couple of days a friend is using ICQ from webpage. I always see him offline, even when he writes me

**Comment by Legodev** [comment]
for me its with all my contacts not just one :( - until yesterday i was one of the few lucky ones that still could use slogin.icq.com, but today i was hit with the need to use icyque.

My mom was hit a few weeks ago, she had the exact same issue, with everyone being offline. I was the only exception, i was marked as online.

**Comment by hitman72ita** [comment]
I wanted to say that I have only one contact now on ICQ :-P

**Comment by kilo6eight** [comment]
Having same problem. All contacts are offline.

**Comment by Siggi0904** [comment]
I think it looks like that contacts from mobile devices ar shown as offline.

**Comment by qtlin** [comment]
Just tired this library in fedora 28. All contacts shown offline, while official client shows one contact online.

**Comment by EionRobb** [comment]
Can someone check: is it just when connecting to the account that no-one shows as online but then they start showing up when the other buddy changes their online/offline/away status?

**Comment by dartraiden** [comment]
Official desktop ICQ 10 client sends "I'm offline" if the user did not interact with the client for 10-15 seconds. He is still working and accepting incoming messages but is displayed as offline.

**Comment by Siggi0904** [comment]
looks like that buddies which are afk (idle) and not online are shown as offline.

**Comment by Demige** [comment]
All contacts is offline

**Comment by Legodev** [comment]
After investigating this matter for a bit i can confirm @dartraiden . As soon as the App is not interacted with e.g. put into the background by opening another App on mobile devices, people are presented as offline. 

The same occurrence as seen in pidgin is also present for the web client provided by icq. 

A work around to prevent any further confusion would be, to read the time for how long they are offline. As long as they are only "offline" for some minutes present them as away.

**Comment by dartraiden** [comment]
Protocol supports last Online (=last active) time. So another solution is to display last online time as status or in popup tips...
